plat/kvm/arm/exceptions.S

```diff
@@ -81,6 +83,12 @@
 	 */
 	msr     tpidrro_el0, x0
 
+	ldr	x0, =ukarch_except_switch_stack
```

Comment:
Should've this been `ur_ldr` instead?
Also nit: I believe you could replace `cmp` + `beq` with a `cbz`:
```
ldr x0, =ukarch_except_switch_stack
cbz x0, 0f                    // Compare and Branch if Zero
```
It could help with branch prediction as this check is, although important, useless unless we are crashing.